volmdlr/core.py

```diff
@@ -666,9 +666,12 @@ def bbox_intersection(self, bbox2):
                 and self.zmin < bbox2.zmax and self.zmax > bbox2.zmin)
 
     def is_inside_bbox(self, bbox2):
-        return (self.xmin > bbox2.xmin and self.xmax < bbox2.xmax \
-                and self.ymin > bbox2.ymin and self.ymax < bbox2.ymax \
-                and self.zmin > bbox2.zmin and self.zmax < bbox2.zmax)
+        return ((self.xmin >= bbox2.xmin or math.isclose(self.xmin, bbox2.xmin, abs_tol=10e-6))\
+                and (self.xmax <= bbox2.xmax or math.isclose(self.xmax, bbox2.xmax, abs_tol= 10e-6))\
+                and (self.ymin >=bbox2.ymin or math.isclose(self.ymin, bbox2.ymin, abs_tol= 10e-6))\
+                and (self.ymax <= bbox2.ymax or math.isclose(self.ymax, bbox2.ymax, abs_tol=10e-6)) \
+                and (self.zmin >= bbox2.zmin  or math.isclose(self.zmin, bbox2.zmin, abs_tol=10e-6))\
+                and (self.zmax <= bbox2.zmax or math.isclose(self.zmax, bbox2.zmax, abs_tol=10e-6)))
```

Comment:
for perf better use a < b + 1e-6
And 10e-6 = 1e-05

Reply:
then we use math.isclose no more?